### PR TITLE
BIG changes!

### PR DESCRIPTION
**Pull request by SouthyOsmanli**
## About The Pull Request
<details>
<summary>
Wanted flash grenades? Can do! 
Player-claw boss sprite? Can do - _without conflict this time!_
How about some new gun sounds for the recently added weapons? Yep!
And, finally, **hunting lodge armor and sprites!**
</summary>
<hr>
</details>

## Changelog
:cl:
- Added: Deathclaw boss sprite for Sieg; located in 64x64 sprites folder.
- Added: Gun sounds for the DP machine gun, M3 Grease Gun, and both PPV/PPSH assault SMGs.
- Added: Flash grenades to the China Lake.
- Added: Hunting lodge armor code and sprites to boot.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/ammunition/bullets.dm [hunk 1]
```diff
@@ -394,21 +394,32 @@
...
 	maxamount = 1
+	matter = list(MATERIAL_STEEL = 2, MATERIAL_PLASTIC = 8)
 
 /obj/item/ammo_casing/grenade/blast
 	name = "blast grenade shell"
-	desc = "An explosive grenade shell, designed to be fired from grenade launchers"
+	desc = "An explosive grenade shell, designed to be fired from grenade launchers."
 	icon_state = "blast"
-	projectile_type = /obj/item/projectile/bullet/grenade
+	projectile_type = /obj/item/projectile/bullet/grenade/
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/ammunition/bullets.dm:403]:
```suggestion
	projectile_type = /obj/item/projectile/bullet/grenade
```

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -551,3 +551,70 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
 	src.icon_state = "reactiveoff"
 	src.item_state = "reactiveoff"
 	..()
+
+/// Hunting Lodge Armor
+
+obj/item/clothing/suit/armor/hunter
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/armor.dm:557]:
```suggestion
/obj/item/clothing/suit/armor/hunter
```

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -551,3 +551,70 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
...
+		melee = 35,
+		bullet = 25,
+		energy = 25,
+		bomb = 10,
+		bio = 0,
+		rad = 0 )
+	body_parts_covered = UPPER_TORSO|LEGS
+	cold_protection = UPPER_TORSO|LEGS
+
+obj/item/clothing/suit/armor/hunter/bone
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/armor.dm:573]:
```suggestion
/obj/item/clothing/suit/armor/hunter/bone
```

File: code/modules/clothing/suits/armor.dm [hunk 4]
```diff
@@ -551,3 +551,70 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
...
+		melee = 45,
+		bullet = 20,
+		energy = 20,
+		bomb = 10,
+		bio = 0,
+		rad = 0 )
+	body_parts_covered = UPPER_TORSO|LEGS
+	cold_protection = UPPER_TORSO|LEGS
+
+obj/item/clothing/suit/armor/hunter/reinforced
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/armor.dm:590]:
```suggestion
/obj/item/clothing/suit/armor/hunter/reinforced
```

File: code/modules/clothing/suits/armor.dm [hunk 5]
```diff
@@ -551,3 +551,70 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
...
+		melee = 30,
+		bullet = 45,
+		energy = 25,
+		bomb = 10,
+		bio = 0,
+		rad = 0 )
+	body_parts_covered = UPPER_TORSO|LEGS
+	cold_protection = UPPER_TORSO|LEGS
+
+obj/item/clothing/suit/armor/hunter/leather
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/armor.dm:606]:
```suggestion
/obj/item/clothing/suit/armor/hunter/leather
```

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Huh, not sure how this even complies

**Comment by SouthyOsmanli** [comment]
Oddly - compiled with that grenades '/' being left out.
The armor probably would not have loaded in right without those changes but due to the lack of the '/' it would compile but the item just would not exist.

Either way - should be good now.

**Comment by SouthyOsmanli** [comment]
...Realized that when the China Lake was fired it would not create an admin log. Fixed.

File: code/modules/projectiles/guns/launcher/projectile_grenade_launcher.dm [hunk 6]
```diff
@@ -1,22 +1,28 @@
...
+	load_method = SINGLE_CASING
+	origin_tech = list(TECH_COMBAT = 6, TECH_MATERIAL = 2)
+	handle_casings = HOLD_CASINGS
+	fire_sound = 'sound/weapons/guns/fire/GLfire.ogg'
+	bulletinsert_sound = 'sound/weapons/guns/interact/batrifle_magin.ogg'   //Placeholder, could use a new sound
+	max_shells = 3
+	recoil_buildup = 20
+	twohanded = TRUE
+
+/obj/item/weapon/gun/launcher/grenade/handle_post_fire(mob/user)
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/launcher/projectile_grenade_launcher.dm:24]:
```suggestion
/obj/item/weapon/gun/projectile/shotgun/pump/china/handle_post_fire(mob/user)
```

**Pull request merged**